db/connect.js

```diff
@@ -1,12 +1,20 @@
-const mongoose = require('mongoose')
+// const mongoose = require('mongoose')
```

Comment:
This is something I would have recommended commenting out as well, good idea. I personally would have added a note comment about how this was the "old way" so future maintainers understand, but this is otherwise a great move.

Reply:
Thank you